inspector/index.html

```diff
@@ -155,17 +155,30 @@
         white-space: nowrap;
       }
 
-      .header-item button {
+      .header-item button,
+      .header-item label {
         cursor: pointer;
         border-radius: 2px;
         margin: 0px 3px;
       }
-      .dark .header-item button {
+
+      .header-item label {
+        font-size: 13px;
```

Comment:
Why a different font-size for label than for button?

Reply:
Yes we could align both explicitly. Would make more sens design-wise and code-wise to me.